src/app/children/child-block/child-block.component.ts

```diff
@@ -22,8 +22,8 @@ export class ChildBlockComponent implements OnInit {
     if (this.entityId !== undefined) {
       this.entityMapper.load(Child, this.entityId).then(child => {
         this.entity = child;
-      }).catch(err => {
-        console.log(err);
+      }).catch(() => {
+        // No special error handling here, as the database will report the technical error and the UI catches the entity being undefined
```

Comment:
As the database errors are silenced for the user, maybe we should display a (user-friendly) alert here?

Reply:
I thought about this, but just handling it silently then seemed like the better option. I hope, you can understand my reasoning (I found it very hard to put it in words):

Imagine this: Someone added a Note about some children. One of these children was then deleted. You could now (in theory) display a message whenever this child's child-block is deleted, i.e. at the Notes Overview. However, there is nothing the end user can do about it except editing the Note and then saving it (as the child-select component won't preselect children that do not exist).

Or you can hide the child-block altogether (which is what is happening now anyways, as we have nothing we could show (neither name nor picture, we have just the (technical) ID)), and don't display a message. When the Note is edited, the problem will solve itself as described above.

Once the child is deleted, there is nothing the user can do other than accepting this -- so why bother him/her? It is, however, important, to inform the user about the consequences of deleting a child in the first place -- but that's a different topic.

Reply:
Yes, I get your point. That's true.
I was thinking along something like: The user reads a note talking about a child and wonders why that child is not referenced. Without a notification the behaviour of the UI is not transparent. But it's probably even more important that - as you point out - the user can't do anything about it anyway.

At the core of this is #220 anyway: this error here should be a fringe case hardly ever occurring.

Go ahead and merge!